src/main/kotlin/app/revanced/patches/grindr/unlimited/fingerprints/IsFreeFingerprint.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.grindr.unlimited.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
+import com.android.tools.smali.dexlib2.Opcode
+
+object IsFreeFingerprint : MethodFingerprint(
+    customFingerprint = { methodDef, _ ->
+        methodDef.name == "r" && methodDef.definingClass.endsWith("storage/s0;")
```

Comment:
This will not work across versions, you should not directly check for obfuscated names.